go/kbfs/libkbfs/keybase_service_base.go

```diff
@@ -1336,6 +1336,21 @@ func (k *KeybaseServiceBase) FinalizeMigration(ctx context.Context,
 	if err != nil {
 		return err
 	}
+	if handle.TypeForKeying() == tlf.TeamKeying {
```

Comment:
Why are we migrating something that is already Team keyed? Or does this handle represent the post-migration state?

Reply:
The handle itself becomes team-based when the service has already 1) created an implicit team for the parties in the folder, and 2) written the TLF ID into that implicit team's sigchain.  The service does this in between calling `StartMigration` and `FinalizeMigration`. You can see that the actual folder migration in `folder_branch_ops` already relies on the handle being team-based: https://github.com/keybase/client/blob/5cf4f306f3d3574fb99fc7fef82362253749c7ac/go/kbfs/libkbfs/folder_branch_ops.go#L7833-L7836

Reply:
Understood, thanks.